Core/Core/Common/CommonUI/CoreWebView/View/CoreWebView.swift

```diff
@@ -729,3 +740,98 @@ extension CoreWebView {
         }
     }
 }
+
+// MARK: - Attachment Download
```

Comment:
Let's extract this logic to a separate file/files/folder. This CoreWebView file is already big enough.